oracle/datadog_checks/oracle/oracle.py

```diff
@@ -88,6 +95,19 @@ def validate_config(self):
         if not self._server or not self._user:
             raise ConfigurationError("Oracle host and user are needed")
 
+        if self._protocol.upper() not in VALID_PROTOCOLS:
+            raise ConfigurationError("Protocol %s is not valid, must either be TCP or TCPS" % self._protocol)
+
+        if self._jdbc_driver and (not self._jdbc_truststore_type or not self._jdbc_truststore_path):
```

Comment:
```suggestion
        if self._jdbc_driver and not (self._jdbc_truststore_type and self._jdbc_truststore_path):
```

Should we use `and` since both are required for this option? also easier to read